react/ProductDetails.js

```diff
@@ -244,15 +257,7 @@ class ProductDetails extends Component {
 
     const highlightsFromAllSpecifications = () => {
```

Comment:
Duplication of line [189](https://github.com/vtex-apps/product-details/pull/110/files#diff-5849199d3a7b18f86218d552920cf88bR189)